data/VidipGhosh.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Vidip Ghosh",
+  "bio": "B.Tech CSE'24 from Jamia Millia Islamia, New Delhi",
+  "tags": ["travelling","photography","exploring new technologies"]
```

```suggestion
  "tags": ["travelling","photography","exploring new technologies"],
```
There is a comma missing here